voluncheer/opportunityboard/test_forms.py

```diff
@@ -0,0 +1,69 @@
+from datetime import timedelta
+
+from django.test import TestCase
+from django.utils import timezone
+
+from opportunityboard.forms.postanopportunity import PostAnOpportunityForm
+from opportunityboard.models import Opportunity
+from profiles.models import Organization
+from profiles.models import User
+from profiles.models import UserType
+
+
+class PostAnOpportunityFormTest(TestCase):
+    """Test cases for Post An Opportunity form"""
+
+    def setUp(self):
+        """See base class."""
+        self.form = PostAnOpportunityForm()
+        self.user = Organization.objects.create(
+            user=User.objects.create(
+                email="jedi@jedi.com",
+                password="peace_and_justice_for_the_galaxy",
+                type=UserType.ORGANIZATION,
+            ),
+            name="Jedi Council",
+        )
+        self.user.save()
+
+    def test_validation(self):
+        """Test Post An Opportunity validation"""
+        data = {
+            "category": "sports",
+            "title": "Jedi Trial",
+            "description": "Face your destiny.",
+            "date": timezone.now(),
+            "duration": timedelta(minutes=20),
+            "address_1": "New York, NY",
+            "address_2": "Mean st",
+            "is_published": True,
+        }
+        form = PostAnOpportunityForm(data=data, instance=self.user)
+        self.assertEqual(self.user.opportunity_set.all().count(), 0)
+        self.assertFalse(form.save())
```

Comment:
What does it mean to assert that `form.save()` is `False`? Same with the line in the below that asserts if `form.update(1)` is `False`.

Reply:
<img width="1027" alt="image" src="https://user-images.githubusercontent.com/43869961/226220869-27fb3e69-4ea2-4c55-88bf-b4fc9bdbde1f.png">

Void function can only be tested using assertFalse